src/utils/db.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('dotenv/config');
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize(
+  'postgres',
+  'postgres',
+  '3562351Boh',

```

Comment:
move this to .env file pls, if u don't understand something, ask for help in **fe_chat**

Reply:
I had everything in the env file, but it didn’t connect to the postrges server through it, but everything works in another task